qdarkstyle/palette.py

```diff
@@ -21,15 +21,15 @@ class DarkPalette(object):
 
     COLOR_SELECTION_LIGHT = '#148CD2'
     COLOR_SELECTION_NORMAL = '#1464A0'
-    COLOR_SELECTION_DARK = '#14506E'
+    COLOR_SELECTION_DARK = '#14506E'"""
```

Comment:
@isabela-pf  you can go ahead and remove those colors as comments, so we have a clean code. :)